Examples/Swift/ViewController.swift

```diff
@@ -115,6 +115,7 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         options.includesSteps = true
         options.routeShapeResolution = .full
         options.profileIdentifier = .automobileAvoidingTraffic
+        options.attributeOptions = [.openStreetMapNodeIdentifier, .speed, .congestionLevel]
```

Comment:
The documentation should be updated to suggest setting `attributeOptions`, especially if congestion levels require all three attribute options (which is undiscoverable).